apps/docs/src/guide/wallets/connectors.md

```diff
@@ -4,6 +4,422 @@
+
+<<< @/../../docs-snippets/src/guide/wallets/connectors.test.ts#fuel-options-storage-memory{ts:line-numbers}
+
+The default behavior will use `LocalStorage` if the `window` is available:
+
+<<< @/../../docs-snippets/src/guide/wallets/connectors.test.ts#fuel-options-storage-local{ts:line-numbers}
+
+#### `targetObject`
+
+The `targetObject` provides a target with which the `Fuel` manager can interact. Used for registering events and can be overridden as follows:
+
+<<< @/../../docs-snippets/src/guide/wallets/connectors.test.ts#fuel-options-target-object{ts:line-numbers}
+
+### Methods
+
+The `Fuel` manager provides several methods to interact with the SDK:
+
+#### All methods from connectors
+
+The `Fuel` manager provides all the [methods](#methods) available from the connected connectors. Thus, you can interact with the current connector as if you were interacting with the `Fuel` manager directly.
+
+If no current connector is available or connected, it will throw an error.
+
+#### `connectors`
+
+The `connectors` method gets the current list of _installed_ and _connected_ connectors.
+
+<<< @/../../../packages/account/src/connectors/fuel.ts#fuel-sdk-method-connectors{ts:line-numbers}
+
+#### `getConnector`
+
+The `getConnector` method resolves a connector by its name. This is useful for finding a specific connector with which to interact. If the connector is not found, it will return `null`.
+
+<<< @/../../../packages/account/src/connectors/fuel.ts#fuel-sdk-method-getConnector{ts:line-numbers}
+
+#### `hasConnector`
+
+The `hasConnector` method will return `true` under the following conditions:
+
+- There is a current connector that is connected.
+- A connector is connected within two seconds of calling the method.
+
+<<< @/../../../packages/account/src/connectors/fuel.ts#fuel-sdk-method-hasConnector{ts:line-numbers}
+
+#### `selectConnector`
+
+The `selectConnector` method accepts a connector name and will return `true` when it is _available_ and _connected_. Otherwise, if not found or unavailable, it will return `false`.
+
+<<< @/../../../packages/account/src/connectors/fuel.ts#fuel-sdk-method-selectConnector{ts:line-numbers}
+
+#### `currentConnector`
+
+The `currentConnector` method will return the current connector that is connected or if one is available and connected, otherwise it'll return `null` or `undefined`.
+
+<<< @/../../../packages/account/src/connectors/fuel.ts#fuel-sdk-method-currentConnector{ts:line-numbers}
+
+#### `getWallet`
+
+The `getWallet` method accepts an address (string or instance) as the first parameter and a provider or network as the second parameter. It will return an `Account` instance for the given address (providing it is valid).
+
+The provider or network will default to the current network if not provided. When a provider cannot be resolved, it will throw an error of code [`INVALID_PROVIDER`](../errors/index.md).
```

Comment:
```suggestion
The provider or network will default to the current network if not provided. When a provider cannot be resolved, it will throw an [`INVALID_PROVIDER`](../errors/index.md) error.